examples/3d/generate_custom_mesh.rs

```diff
@@ -51,16 +51,15 @@ fn setup(
         Transform::from_xyz(1.8, 1.8, 1.8).looking_at(Vec3::ZERO, Vec3::Y);
 
     // Camera in 3D space.
-    commands.spawn(Camera3dBundle {
+    commands.spawn((Camera3dBundle {
         transform: camera_and_light_transform,
         ..default()
-    });
+    },));
```

Comment:
Unnecessary tuple (I thought clippy would pick this up?)

Reply:
Clippy doesn't pick it up because `(T,)` and `T` have different `Bundle` implementations.

Reply:
I see, thx